cmd/import_test.go

```diff
@@ -47,6 +49,20 @@ field = "f1"
 				return v.Error()
 			},
 		},
+		{
+			args: []string{"import", "--index", "i1", "--field", "f1", "--field-keys", "--field-max", "100"},
```

Comment:
I'd like to see tests that use all the new flags - especially time quantum since that one is a bit weird.

Reply:
Updated